app/tests/unit/containers/Applications/ApplicationWizard.test.tsx

```diff
@@ -1,8 +1,50 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 import {ApplicationWizardComponent} from 'containers/Applications/ApplicationWizard';
+import {ApplicationWizard_query} from 'ApplicationWizard_query.graphql';
 import ViewApplication from 'pages/reporter/application/[applicationId]/version/[versionNumber]/view';
 
+const getTestQuery = () => {
```

Comment:
I've started this pattern in #1574 as well - using a factory function to define test queries because it was such a pain to update them all.

I want to ideally update in one place next time!  🤪